testutil/singlevm/setup.sh

```diff
@@ -79,6 +118,33 @@ done
 
 set -o nounset
 
+echo "Generating configuration file $conf_file"
+(
+cat <<-EOF
+configure:
+  scheduler:
+    storage_uri: /etc/ciao/configuration.yaml
+  storage:
+    ceph_id: ciao
+  controller:
+    compute_ca: ${ciao_pki_path}/${ciao_controller_cert}
+    compute_cert: ${ciao_pki_path}/${ciao_controller_key}
+    identity_user: ${ciao_username}
+    identity_password: ${ciao_password}
+  image_service:
+    type: glance
+    url: https://${ciao_host}
+  launcher:
+    compute_net: [${ciao_subnet}]
+    mgmt_net: [${ciao_subnet}]
+    disk_limit: false
+    mem_limit: false
+  identity_service:
+    type: keystone
+    url: https://${ciao_host}:${keystone_ext_port}
+EOF
+) > $conf_file
```

Comment:
Should this file be stored under ~/local along with everything else?  Right now it's being created in my ciao source folder.